test/utils/helpFn.ts

```diff
@@ -67,29 +146,37 @@ export const sendExtrinsic = async (
       polkadotApi,
```

Comment:
Yes, it's better to pass the `keyringPair` into the `sendExtrinsic` function to sign and dispatch the extrinsic. 

Currently, within the code structure, the signing step is already completed during the extrinsic build process. 

This might be something to consider for modification during future code refactoring.